docs/api/api-spec.md

```diff
@@ -132,6 +132,34 @@ On the financial desktop, applications often want to broadcast context to any nu
 ## Resolvers
 Intents functionality is dependent on resolver functionality to map the intent to a specific App.  This will often require end-user input.  Resolution can either be performed by the Desktop Agent (raising UI to pick the desired App for the intent) or by the app launching the intent - in which case the calling App will handle the resolution itself (using the findIntents API below) and then invoke an explicit Intent object.
 
+## Context channels
+
+The context channel api allows a set of apps to share a stateful piece of data between them, and be alerted when it changes.
+
+There are three types of channels, which are functionally identical, but have different visibility and discoverability semantics.
+
+1. The 'public' or 'desktop' ones, which have a well understood identity. One is called 'default'.
+2. The 'private' or 'app' ones, which have a transient identity and need to be revealed
+
+The 'public' channels include a 'default' channel which serves as the backwards compatible layer with the 'send/broadcast context' above. There are some reserved channel names for future use. Currently this is just 'global'.
```

Comment:
> There are some reserved channel names for future use. Currently this is just 'global'.

For this to be workable, _all_ reserved words need to be declared at the point this API is released, otherwise application developers aren't able to avoid naming collisions.